packages/key-did-resolver/src/nist_weierstrass_common.ts

```diff
@@ -8,8 +8,8 @@ import { base64url } from 'multiformats/bases/base64'
   * For BigInt see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt
   */
 interface BigIntPoint {
-   x: BigInt,
-   y : BigInt
+   x: bigint,
+   y : bigint
```

Comment:
Why this change?

Reply:
Why is your linter different from the rest of the teams?

Reply:
I don't understand why this change is needed based on the changes you did in the code.